calc/async_calc_graph.go

```diff
@@ -187,6 +187,7 @@ func (acg *AsyncCalcGraph) onEvent(event interface{}) {
 
 func (acg *AsyncCalcGraph) Start() {
 	log.Info("Starting AsyncCalcGraph")
-	acg.flushTicks = time.Tick(tickInterval)
+	flushTicker := time.NewTicker(tickInterval)
+	acg.flushTicks = flushTicker.C
```

Comment:
1. After pondering our CI failures, I believe this should be `NewTicker`, not `NewTimer`.

2. What motivated this change anyway?